spanner_orm/tests/query_test.py

```diff
@@ -59,6 +59,23 @@ def test_count(self, sql_query):
     self.assertEqual({column_key: value}, parameters)
     self.assertEqual(types, {column_key: field.Integer.grpc_type()})
 
+  @mock.patch('spanner_orm.table_apis.sql_query')
+  def test_count_allows_force_index(self, sql_query):
+    sql_query.return_value = [[0]]
+    column, value = 'int_', 3
+    equal_to = condition.equal_to(column, value)
+    force_index = condition.force_index('test_index')
+    models.UnittestModel.count(True, equal_to, force_index)
+    (_, sql, parameters, types), _ = sql_query.call_args
+
+    column_key = '{}0'.format(column)
+    expected_sql = (
+        r'SELECT COUNT\(\*\) FROM table@{{FORCE_INDEX=test_index}} '
+        r'WHERE table.{} = @{}').format(column, column_key)
```

Comment:
this test should just be about the FORCE_INDEX directive, so the amount of code here can be reduced. see the other test for force index in this file for guidance